src/sentry/conf/server.py

```diff
@@ -3022,12 +3010,8 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
 }
 
 
-class TopicDefinition(TypedDict):
-    cluster: str
-
-
 # Cluster configuration for each Kafka topic by name.
-KAFKA_TOPICS: Mapping[str, Optional[TopicDefinition]] = {
+KAFKA_TOPICS: Mapping[str, Optional[sentry.utils.kafka_config.TopicDefinition]] = {
```

Comment:
I've been trying to reduce the app imports in this file -- any imports in here tend to cause cycles which turns off type checking for settings annoyingly.  they're really tough to track down as well :(

Reply:
this used to be a string-typed annotation `Optional["sentry.utils..."]`, but the formatter ripped it apart again

Isn't `from __future__ import annotations` supposed to deal with this anyway?

Reply:
the problem is more that the type is defined outside the settings file than the quoting part

for some reason django-stubs cannot resolve cycles at all in the settings file -- even if they are legal

Reply:
should we have a types.py for types used in settings?

i need type definitions to be available in both the settings file and in app code

Reply:
yeah that would probably work